### ISSUE TITLE
Support for dpdk

### ISSUE THREAD

**Issue by MatiasVara** [eclipse-cyclonedds/cyclonedds#1914]
Hello, 

apologies if this question has already been asked, is it possible to run CycloneDDS with DPDK? 

Thanks,

**Comment by eboasson** [comment]
DPDK as in https://dpdk.org ? The briefest of summaries from wikipedia suggests it would be really valuable to support it, but I have no idea what it takes to make it run with DPDK without looking at it more closely ... maybe a lot, maybe there's a trivial trick ...

**Comment by MatiasVara** [comment]
> DPDK as in https://dpdk.org ? The briefest of summaries from wikipedia suggests it would be really valuable to support it, but I have no idea what it takes to make it run with DPDK without looking at it more closely ... maybe a lot, maybe there's a trivial trick ...

Yes, that one. I think the use of DPDK could improve a lot latency and throughput.

**Comment by wadehunk** [comment]
Discovered this recently. No idea about the quality, functionality, or if there has been any mention of submitting a PR for it: https://github.com/caps-tum/cyclonedds-dpdk-xdp
